client/src/components/Sprint/SprintIssues/index.tsx

```diff
@@ -2,51 +2,99 @@ import React from 'react';
+}
+interface DragProps {
+	listId: string;
+	listType?: string;
+	internalScroll?: boolean;
+	isCombineEnabled?: boolean;
+}
 
-export const SprintIssues: React.FC<Props> = ({ issues, noIssuesText }: Props) => {
-	const issuesList = !!issues?.length ? (
-		issues.map((issue) => {
-			const { type, priority } = issue;
-			return (
-				<List.Item href={`/issue/${issue.issueKey}`} key={issue.id} className={styles.issueItem}>
-					<List.Content className={styles.leftContent}>
-						<List.Icon name={type?.icon as any} color={type?.color as any} title={type?.title} />
-						<List.Header>{issue.summary}</List.Header>
-					</List.Content>
-					<List.Content className={styles.rightContent}>
-						<AssigneeAvatar user={issue.assigned} />
-						<Item
-							className={
-								issue.status?.title === 'Done' ? styles.issueKeyItemCompleted : styles.issueKeyItem
-							}
-						>
-							{issue.issueKey}
-						</Item>
-						<List.Icon
-							title={priority?.title}
-							name={priority?.icon as any}
-							color={priority?.color as any}
-							className={styles.priorityItem}
-						/>
-					</List.Content>
-				</List.Item>
-			);
-		})
-	) : (
-		<List.Item className={styles.emptyIssueItem}>
-			<span>{noIssuesText}</span>
-		</List.Item>
-	);
+export const SprintIssues: React.FC<Props & DragProps> = (props: Props & DragProps) => {
+	const { t } = useTranslation();
+
+	const renderIssues = (dropProvided: DroppableProvided) => {
+		return (
+			<div ref={dropProvided.innerRef}>
+				<List
+					selection={!!props.issues?.length}
+					verticalAlign="middle"
+					style={{ marginTop: '14px', marginBottom: '14px' }}
+				>
+					{!!props.issues?.length ? (
+						props.issues.map((issue, index) => (
+							<Draggable key={issue.id} draggableId={issue.id} index={index}>
+								{(dragProvided) => (
+									<div
+										{...dragProvided.dragHandleProps}
+										{...dragProvided.draggableProps}
+										ref={dragProvided.innerRef}
+									>
+										<List.Item
+											href={`/issue/${issue.issueKey}`}
+											key={issue.id}
+											className={styles.issueItem}
+										>
+											<List.Content className={styles.leftContent}>
+												<List.Icon
+													name={issue.type?.icon as any}
+													color={issue.type?.color as any}
+													title={issue.type?.title}
+												/>
+												<List.Header>{issue.summary}</List.Header>
+											</List.Content>
+											<List.Content className={styles.rightContent}>
+												<AssigneeAvatar user={issue.assigned} />
+												<Item
+													className={
+														issue.status?.title === 'Done'
+															? styles.issueKeyItemCompleted
+															: styles.issueKeyItem
+													}
+												>
+													{issue.issueKey}
+												</Item>
+												<List.Icon
+													title={issue.priority?.title}
+													name={issue.priority?.icon as any}
+													color={issue.priority?.color as any}
```

Comment:
Why any? Удали any, если можешь

Reply:
Can't be changed since `name` and `color` prop of `<Icon>` in semantic have predefined types, while we save icon as string type.

```
Type 'string' is not assignable to type "code" | "header" and so on ...
```

```
(JSX attribute) StrictIconProps.name?: "code" | "header" | "map" | ... 1235 more ... | undefined
```